lua/cfc_time/server/storage_options/mysql.lua

```diff
@@ -128,28 +128,31 @@ function storage:PlayerInit( ply, sessionStart, callback )
     local steamID = ply:SteamID64()
 
     logger:debug( "Receiving PlayerInit call for: " .. tostring( steamID ) )
-    local transaction = storage:InitTransaction()
+    local lastSessionQuery = self:Prepare( "lastSession", nil, steamID )
```

Comment:
Because transactions aren't guaranteed to complete in order, we get the last session (if it exists) first, then handle the `newUser` + `newSession` transaction.